helpers/queries/cow-protocol.sql

```
@@ -1,29 +1,56 @@
+        and call_block_time < date_add('day', 1, (select start_date from day_selector))
+        and call_block_time < date('2025-11-01')
     group by 1
 )
+, mevblocker_sale as (
+    select *
+    from (
+        values
+        (timestamp '2025-11-04', 0.5*6908146.85, 0.5*6908146.85/3212.33)
+        ,(timestamp '2026-01-21', 0.5*44132.69, 0.5*14.9)
+        ,(timestamp '2026-02-17', 0.5*169597.29, 0.5*82.7705679057619)
+    ) as t(date, mev_blocker_sale_usd, mev_blocker_sale_eth)
```

SQLFluff (4.2.1)

[error] Alias 't' is never used in SELECT statement.

(AL05)